src/main.cpp

```diff
@@ -267,38 +234,32 @@ int main(int argc, char* argv[])
         return 0;
     }
 
-    /*******************************************************************
-     * If we get to HERE then we are going to run a GUI application... *
-     *******************************************************************/
-
-#if defined(Q_OS_WIN32) && defined(INCLUDE_UPDATER)
-    auto abortLaunch = runUpdate();
-    if (abortLaunch) {
+    if (parser.isSet(showVersion)) {
+        // Do "version" action - wording and format is quite tightly specified by the coding standards
+#if defined(QT_DEBUG)
+        texts << QCoreApplication::translate("main", "%1 %2%3 (with debug symbols, without optimisations)\n",
+         "%1 is the name of the application like mudlet or Mudlet.exe, %2 is the version number like 3.20 and %3 is a build suffix like -dev")
+                 .arg(QLatin1String(APP_TARGET), QLatin1String(APP_VERSION), QLatin1String(APP_BUILD));
+#else // ! defined(QT_DEBUG)
+        texts << QLatin1String(APP_TARGET " " APP_VERSION APP_BUILD " \n");
+#endif // ! defined(QT_DEBUG)
+        texts << QCoreApplication::translate("main", "Qt libraries %1 (compilation) %2 (runtime)\n",
+             "%1 and %2 are version numbers").arg(QLatin1String(QT_VERSION_STR), qVersion());
+        texts << QCoreApplication::translate("main", "Copyright © 2008-%1  Mudlet developers\n").arg(QStringLiteral(__DATE__).mid(7, 4));
```

Comment:
```suggestion
        texts << QCoreApplication::translate("main", "Copyright © 2008-2020  Mudlet developers\n");
```

I know that's how we had it before but for the idea of reproducible builds, https://reproducible-builds.org it's better to hardcode it and change once a year.

Reply:
:information_source: There is a `// PLACEMARKER: Date-stamp needing annual update` comment in two *other* places in the original code which was put there to make them easy to find (with the Qt Creator "ToDo"  plugin) once per year - somehow it seems I had missed THAT one...